torchci/test/autoLabelBot.test.ts

```diff
@@ -317,6 +317,37 @@ describe("auto-label-bot", () => {
     scope.done();
   });
 
+  test("custom repo labels get add when on a matching repo and file", async () => {
+
+    nock("https://api.github.com")
+      .post("/app/installations/2/access_tokens")
+      .reply(200, { token: "test" });
+
+    const payload = requireDeepCopy("./fixtures/pull_request.opened")["payload"];
+    payload["pull_request"]["title"] = "modify a pytorch/tau file";
```

Comment:
isn't this a fake-test-repo file?
```suggestion
    payload["pull_request"]["title"] = "modify a pytorch/fake-test-repo file";
```